index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="new project for microverse">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Ubuntu:wght@300&display=swap"
+        rel="stylesheet">
+    <link href="<link rel=" preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Kanit:wght@600&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@200;400&family=Ubuntu:wght@300&display=swap"
+        rel="stylesheet">
+    <title>My portfolio project</title>
+</head>
+
+<body>
+<!--     <div>
+        <img id="backgroundimg" src="/img/shapes.svg" alt="background">
+    </div> -->
+    <header id="navbar">
+        <nav id="menubar">
+            <a href="index.html" id="logo">My Logo</a>
+            <img id="menu" src="/img/menu.svg" alt="menu-button">
+        </nav>
+        
+    </header>
+
+    <main id="mainCont">
+        <h1 id="name">
+            Damian M. <br> Dib Bustros
+        </h1>
+       
+                <p id="introduction">
+            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my
+            work. <br>
+             <br> If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+
+        <div class="buttons">
+            <a href="#" class="submit-button">Start Collaboration</a>
+        </div>
+
+    </main>
+
+    <footer>
+    </footer>
+</body>
```

Comment:
Kindly implement the changes required for this milestone which is the projects section of the portfolio. So far, you have only implemented the headline section.

Your design:
<div align='center'><img src='https://user-images.githubusercontent.com/20554583/219923385-3072a16f-2c3b-446d-b66a-3e4795c13b98.png' /></div>

Section to implement:
<div align='center'>
<img src='https://user-images.githubusercontent.com/20554583/219923422-49883b55-7511-4d12-a6cd-ca69ae48d832.png' />
<img src='https://user-images.githubusercontent.com/20554583/219923431-86afba02-3661-4b8b-9332-d7ad9a319c5c.png' />
</div>